### DISCUSSION TITLE
Doubts in Backend and usage of the submitted projects

### DISCUSSION THREAD

**Discussion by GipsonRahul** [microsoft/hack-together-teams#46] [Q&A]
Hi all, I have doubt in how to use Azure as Backend for Teams Application. And how to give access to the reviewer (who review our subm itted project) to use the Teams Application

**Comment by waldekmastykarz** [comment]
@GipsonRahul, we'll be reviewing the video you'll share with us as part of your submissions as well as the code of your hack that you'll share with us by publishing your repo. We don't need access to your Azure subscription.

Could you elaborate a bit more what specifically related to using Azure as backend are you concerned about?
